source/core/parameters_lagrangian.cc

```diff
@@ -1039,6 +1096,8 @@ namespace Parameters
 
     template class PhysicalProperties<2>;
     template class PhysicalProperties<3>;
+    template class ForcesNdTorques<2>;
+    template class ForcesNdTorques<3>;
```

Comment:
Not a huge fan of the class name. Something more appropriate like ForcesTorquesOnWall or something would be better.

Reply:
Will change that to ForcesTorquesOnWall